### ISSUE TITLE
This particular website doesn't yield to Anti-Adblock Killer

### ISSUE THREAD

**Issue by dpaks** [reek/anti-adblock-killer#1786]
Website: http://timesofindia.indiatimes.com/top-stories/Supreme-Court-gives-Indian-cricketers-a-voice/articleshow/53277434.cms

**Comment by applesam** [comment]
Intall Tampermonky Beta  use  this  script  >>>>>>>https://x01x012013.github.io/AdBlockProtector/ Install this script.......

**Comment by X01X012013** [comment]
Yes, Times Of India is fixed in my script, you can use it while waiting for Aak to update.
